ZelBack/src/services/appsService.js

```diff
@@ -9308,21 +9329,43 @@ async function checkAndRemoveApplicationInstance() {
       // eslint-disable-next-line no-await-in-loop
       const runningAppList = await getRunningAppList(installedApp.name);
       const minInstances = installedApp.instances || config.fluxapps.minimumInstances; // introduced in v3 of apps specs
-      if (runningAppList.length > (minInstances + config.fluxapps.maximumAdditionalInstances)) {
```

Comment:
I think its fine to remove BUT
- By not having this flag, any small deviances in the network can cause more installations and uninstallations.
If my node does not receive some app running message just from one node, it will install this application. And then it will uninstall it again once it received new message. I think having this +1 we allow for one missing app runnning message which is effectively resulting in less random installs/uninstalls. 
Just a note to observe the network after this removal.